src/dataTypes/Dashboard/DashboardTypes.ts

```diff
@@ -0,0 +1,14 @@
+export type DashboardItemInfo = {
+    size: DashboardItemSize;
+    withText: boolean;
+};
+
+export enum DashboardItemSize {
+    small,
+    medium,
+    large,
```

Comment:
```suggestion
    SMALL = 2,
    MEDIUM = 3,
    LARGE = 4,
```

Default is all-caps for enums.
Set the values for the enum values here instead of retrieving them later.